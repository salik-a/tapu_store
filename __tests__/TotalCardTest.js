import React from 'react'
import { fireEvent, render, waitFor } from '@testing-library/react-native'
import Provider from "../src/context/provider"

import TotalCard from '../src/components/totalCard/TotalCard';
test('should match with snapshot', () => {

    const rendered = render(
        <Provider><TotalCard /></Provider>
    );
    expect(rendered).toMatchSnapshot();
});
