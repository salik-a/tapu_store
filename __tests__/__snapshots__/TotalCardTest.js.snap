// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match with snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "marginBottom": 50,
        "paddingHorizontal": 35,
        "paddingVertical": 10,
      },
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "black",
          "fontSize": 24,
          "fontWeight": "700",
          "marginVertical": 10,
        },
      ]
    }
  >
    Ürünlerin Toplamı:
  </Text>
  <Text
    style={
      Array [
        Object {
          "color": "black",
          "fontSize": 18,
          "fontWeight": "200",
          "marginVertical": 5,
        },
      ]
    }
  >
    Toplam: 
    0
     TL
  </Text>
  <Text
    style={
      Array [
        Object {
          "color": "black",
          "fontSize": 18,
          "fontWeight": "200",
          "marginVertical": 5,
        },
      ]
    }
  >
    Vergiler + Kargo: 
    0
     TL
  </Text>
  <Text
    style={
      Array [
        Object {
          "color": "black",
          "fontSize": 22,
          "fontWeight": "700",
          "marginVertical": 6,
        },
      ]
    }
  >
    Genel Toplam: 
    0
     TL
  </Text>
</View>
`;
